dlpy/model.py

```diff
@@ -690,14 +1046,19 @@ def train(self, table, attributes=None, inputs=None, nominals=None, texts=None,
         if best_weights is not None:
             b_w = dict(replace=True, name=best_weights)
 
+        if not tensorboard:
```

Comment:
Is it sufficient to to use "not tensorboard" here?  Do TensorBoard objects respond to boolean tests like this, or are you just testing it for None here?